src/js/view/controls/components/timeslider.ts

```diff
@@ -304,8 +304,9 @@ class TimeSlider extends Slider {
             // timeTip may go outside the bounds of the player. Determine the % of tolerance needed
             timeTipPct = (timeTipWidth - tolerance) / (2 * 100 * widthPct);
         }
-        const safePct: number = parseFloat(Math.min(1 - timeTipPct, Math.max(timeTipPct, pct)).toFixed(3)) * 100;
-        transform(timeTip.el, `translateX(${safePct}%)`);
+        const safePct: number = parseFloat(Math.min(1 - timeTipPct, Math.max(timeTipPct, pct)).toFixed(3));
+        const safePos = safePct * railBounds.width;
+        transform(timeTip.el, `translateX(${safePos}px)`);
```

Comment:
If you get rid of `const widthPct = railBounds.width / 100;` from the equations above then calculate `timeTipPixels` instead of `timeTipPct` all the math will be in pixels.

Reply:
Something like
```
   timeTipPixels = (timeTipWidth - tolerance) / 2;
}
const safePixels = parseFloat(Math.min(1 - timeTipPixels, Math.max(timeTipPixels, pct * railBounds.width)).toFixed(3));
```